frontend/package-lock.json

```diff
@@ -13,7 +13,7 @@
                 "@carbon/themes": "^11.43.0",
                 "@carbon/type": "^11.31.0",
                 "@tanstack/vue-query": "^5.56.2",
-                "aws-amplify": "^5.3.27",
+                "aws-amplify": "^6.15.0",
```

Comment:
I'm not sure why there is still aws-amplify changes in the package-lock.json, I thought it should be captured in the last pr. But I tried, everything works fine.